content/blog/insights-cloud-account-discovery/index.md

```diff
@@ -0,0 +1,95 @@
+
+<!--more-->
+
+### Account Management
+
+To get started, Insights Account Discovery provides a simple UI approach to setting up all of your infrastructure accounts to be scanned and read into the Insights supergraph. Once discovered, all of your infrastructure resources can then be seen in the Resource Explorer and accessed by Pulumi CoPilot and Resource Search, providing you the tools needed to deliver on your strategic infrastructure goals.
+
+Account Discovery starts with the new Accounts page to provide a list of all accounts created along with the current status of the latest account scans. The Accounts page is the single place to go to manage all aspects of the Insights Account Discovery process.
+
+{{< video title="Accounts" src="accounts.mp4" autoplay="true" loop="true" >}}
+
+### Account Creation
+
+From this page you can create new Accounts to provide the configuration and credentials needed for Insights to regularly scan and manage your infrastructure. Once you create a new top level Account, Insights will automatically create child accounts, based on the underlying platform model, for each group you enabled. These child accounts enable you to control the discovery behavior for each group separately. For example, AWS enables you to divide you infrastructure into regions and Insights will create separate child accounts for each region you specify.
+
+{{% notes type="info" %}}
+Account Discovery leverages Pulumi ESC to enter and manage the credentials needed for the Discovery process to find and read all of the infrastructure resources. By relying on ESC, Insights aligns with enterprise best practices for creating and handling application secrets.
+{{% /notes %}}
+
+{{< video title="Create Account" src="create-account.mp4" autoplay="true" loop="true" >}}
+
+### Understanding Your Infrastructure
+
+Once each of your Accounts have started scanning, your resources will be displayed in the Insights Resources Explorer along with all Infrastructure as Code resources that you have imported. The Resources Explorer and Resources Search enables you to ask questions and factor your resources into logical groupings that significantly improves the process for managing your infrastructure and getting answers to key questions needed for your projects. For example, you can ask the AI Resource Search to 'Find all public IP addresses' and any other question you might have about your infrastructure.
+
+{{< video title="Discovered Resources" src="resources.mp4" autoplay="true" loop="true" >}}
+
+### Conclusion
```

Comment:
There are two areas I think we should speak to before concluding:

1/ Policy. This is a fast follow, right? Should we at least make mention of it? Or will we just blog about it separately when it's available?

2/ Pricing. Should we say, "Free while in public preview, with per-resource pricing coming in Q1 2025"? People are often nervous to use something if they don't know how it will be priced.

Reply:
I talked about Policy in the last line of the conclusion. However I created a new section called 'More to Come' and moved this into that section:

_We are excited about the potential of Pulumi Insights and are working hard to bring additional capabilities to continue to simplify the experience of managing today's complex cloud environments. Watch for additional features like Pulumi Crossguard Policy support for Discovered resources, along with additional platform support, and more, coming soon._

_Insights Account Discovery is free while in public preview, with per-resource pricing for Team, Enterprise and Business Critical tiers coming in Q1 2025._